### DISCUSSION TITLE
How to see results in the UI for runs that were executed from the CLI

### DISCUSSION THREAD

**Discussion by takerfume** [dagster-io/dagster#17704] [Q&A] [answered]
During the development phase, I'd like to run jobs with various parameters to see which ones cause asset materialization to fail, and what the metadata for those materializations looks like. 

Is there a way to view the results of job executions from the dagster job execute command or a Python script within the Dagster development UI?

 Being able to do this would significantly streamline my experimentation process.

**Comment by sryza** [comment, marked as answer]
There are two processes at play here:
- The webserver process
- The CLI invocation

If you make sure that the DAGSTER_HOME environment variable points to the same directory in the environments of both of these processes, then activity that occurs during the CLI invocation will be displayed in the UI.

**Reply by takerfume** [reply]
sorry for late reply, but your advice works well.

Thank you!

**Comment by sryza** [comment]
Reopened discussion because closed discussions don't appear in search results
